sandbox/grommet-app/src/pages/layouts/kinds/HPEMarketing/HPEProducts.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useContext } from 'react';
+import {
+  Box,
+  Button,
+  Grid,
+  Heading,
+  PageContent,
+  ResponsiveContext,
+  Tabs,
+  Tab,
+  Image,
+} from 'grommet';
+import { LinkNext } from 'grommet-icons';
+import { CardTemplate } from './components';
+
+export const HPEProducts = () => {
+  const breakpoint = useContext(ResponsiveContext);
+  const [index, setIndex] = React.useState(2);
+
+  const columns =
+    breakpoint === 'xsmall'
```

Comment:
Should we also go single column at small? seems like this is too tight.

<img width="783" alt="Screenshot 2025-03-20 at 10 39 43 AM" src="https://github.com/user-attachments/assets/0230d220-49b3-417e-938f-6fec048f576d" />